From c54a951bf04474a58be3ee4fe479839efa409868 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Tue, 2 Aug 2016 17:35:01 +0000
Subject: [PATCH] OS-5569 uberdata32_t is missing ul_brand member Reviewed by:
 Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 usr/src/lib/libc/inc/thr_uberdata.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/lib/libc/inc/thr_uberdata.h b/usr/src/lib/libc/inc/thr_uberdata.h
index ff18c8e9f4..ac7af4ce29 100644
--- a/usr/src/lib/libc/inc/thr_uberdata.h
+++ b/usr/src/lib/libc/inc/thr_uberdata.h
@@ -1185,6 +1185,7 @@ typedef struct uberdata32 {
 	caddr32_t	robustlocks;
 	caddr32_t	robustlist;
 	caddr32_t	progname;
+	caddr32_t	ub_broot;
 	caddr32_t	ub_comm_page;
 	caddr32_t	tdb_bootstrap;
 	tdb32_t		tdb;
-- 
2.21.0

